tests/unit/test_unit_smhi.py

```diff
@@ -0,0 +1,249 @@
+"""SMHI unit tests."""
+
+from unittest.mock import MagicMock, patch
+
+import pandas as pd
+import pytest
+from smhi.smhi import SMHI
+
+
+class TestUnitSMHI:
+    """Unit tests for SMHI class."""
+
+    @patch("smhi.metobs.Parameters.__new__")
+    def test_unit_smhi_init(self, mock_parameters):
+        """Unit test for SMHI init method.
+
+        Args:
+            mock_requests_metobs: mock requests metobs object
+        """
+        client = SMHI()
+
+        assert client.parameters
+
+    @pytest.mark.parametrize("parameter", [(None), (1)])
+    @patch("smhi.metobs.Stations.__new__")
+    def test_unit_smhi_get_stations(self, mock_station_data, parameter):
+        """Unit test for SMHI get_stations method.
+
+        Args:
+            parameter: parameter (int)
+        """
+        client = SMHI()
+        assert client.get_stations(parameter)
```

Comment:
Same here.

Reply:
This would have failed if the method didn't return a value. Since Parameters is mocked, it will essentially accept anything as long as get_stations accepts an input and returns an output, which it does.